integration-tests/wallet-empty-delegated-implicit-account.spec.ts

```diff
@@ -4,13 +4,13 @@ CONFIGS().forEach(({ lib, rpc, setup, createAddress, knownBaker }) => {
   const Tezos = lib;
 
 
-  describe(`Test emptying a delegated implicit wallet account using: ${rpc}`, () => {
+  describe(`Test emptying a delegated implicit account through wallet api using: ${rpc}`, () => {
 
     beforeEach(async (done) => {
       await setup()
       done()
     })
-    test('creates a new wallet account, delegates it, and attempts to empty it despite the delegation being expected to fail', async (done) => {
+    test('Verify that new Account can be created, delegated and attempt to empty, it should fail despite delegation through wallet api', async (done: () => void) => {
```

Comment:
Consider removing ` through wallet api` because it is already in the `describe`

Reply:
changed